### PR TITLE
Fix grammar and wording

### PR DESCRIPTION
**Pull request by LorenzoSapora**
- Capitalising Scavenger across the whole readme
- Fixed typos
- Some minor sentence structure fixes

### CODE REVIEW

File: readme.md [hunk 1]
```diff
@@ -276,18 +276,18 @@ The `targets` array is to contain a list of entities (to be scraped from) keyed
...
 
 ## Glossary of Terms
 The following words may appear in context above.
 
-- `Daemon`: User instance to be used by the scavenger service.
+- `Daemon`: User instance to be used by the Scavenger service.
 - `Scrap`: Scraped data before being converted to the target object.
 - `Target`: Configured source-model mapping for a single entity. 
-- `Target Object`: Eloquent model object to be generated from scrap. 
+- `Target Object`: Eloquent model object to be generated from scrape.
```
Comment (reliq) [inline readme.md:287]:
```suggestion
- `Target Object`: Eloquent model object to be generated from scrap. 
```

This is referring to a "scrap", as defined on line [285](https://github.com/reliqarts/laravel-scavenger/pull/6/files#diff-5a831ea67cf5cf8703b0de46901ab25bd191f56b320053be9332d9a3b0d01d15R285)

File: readme.md [hunk 2]
```diff
@@ -276,18 +276,18 @@ The `targets` array is to contain a list of entities (to be scraped from) keyed
 - `dissect`: Split compound attributes into smaller attributes via REGEX. (optional)
 - `preprocess`: Array of attributes which need to be preprocessed. `[attributeName => callable]` (optional)
 - `remap`: Array of attributes which need to be renamed in order to be saved as target objects. `[attributeName => newName]` (optional)
-- `bad_words`: Any scraps found containing these words will be discarded. (optional)
+- `bad_words`: Any scrapes found containing these words will be discarded. (optional)
```
Comment (reliq) [inline readme.md]:
```suggestion
- `bad_words`: Any scrap found containing these words will be discarded. (optional)
```

This is referring to a scrap as defined on line [285](https://github.com/reliqarts/laravel-scavenger/pull/6/files#diff-5a831ea67cf5cf8703b0de46901ab25bd191f56b320053be9332d9a3b0d01d15R285)

File: readme.md [hunk 3]
```diff
@@ -68,12 +68,12 @@ Scavenger provides the following features and more out-the-box.
...
 ## Configuration
 
-Scavenger is highly configurable. These configurations remain for use the next time around. 
+Scavenger is highly configurable. Once configured, the settings will be used for every scrape.
 
 
 ### Structure
 
-Below is an example of a typical config file structure, with explaining comments.
+Below is an example of a typical config file structure, with comments explaining the settings.
```
Comment (reliq) [inline readme.md]:
```suggestion
Below is an example of a typical config file structure, with comments explaining each setting.
```

**Review by reliq — CHANGES_REQUESTED** [review]

File: readme.md (hunk 1 above)
Comment (LorenzoSapora) [reply readme.md:287]: But "scrap" is "a scrap of paper".  And "scrape" is "I am going to scrape that webpage"

File: readme.md (hunk 2 above)
Comment (LorenzoSapora) [reply readme.md]: See comment above

**Comment by LorenzoSapora** [comment]
Made the changes, but have some issues that need to be resolved.

File: readme.md (hunk 1 above)
Comment (reliq) [reply readme.md:287]: Makes sense, however a "scrap" is defined in this context as:

`Scraped data before being converted to the target object.`

"Scrap" is used consistently throughout the package to have this meaning.

Comment (LorenzoSapora) [reply readme.md:287]: Sure, but that's a failure in naming, and not the fault of the README. It really should be `scrape`. However, I won't push the point any further, and accept your suggestions.

**Comment by LorenzoSapora** [comment]
You need to fix your composer.json to make these tests pass.

**Review by reliq — APPROVED** [review]

**Comment by reliq** [comment]
> You need to fix your composer.json to make these tests pass.

The fix is on the main branch. Please update your branch from main and I will merge this in. 

Apologies for the late reply and thank you for your contribution.

**Comment by LorenzoSapora** [comment]
Done

**Pull request merged**